examples/Utils/ChromeTraceUtil.cpp

```diff
@@ -184,7 +184,7 @@ void MyEnterProfileZoneFunc(const char* msg)
 		return;
 #ifndef BT_NO_PROFILE
 	int threadId = btQuickprofGetCurrentThreadIndex2();
-	if (threadId<0)
+	if (threadId<0 || threadId >= BT_QUICKPROF_MAX_THREAD_COUNT)
```

Comment:
Did you run it on a CPU with >64 threads? Or does the thread index grow over 64 for other reasons?